platform/windows/display_server_windows.cpp

```diff
@@ -2409,8 +2409,8 @@ void DisplayServerWindows::set_icon(const Ref<Image> &p_icon) {
 		SendMessage(windows[MAIN_WINDOW_ID].hWnd, WM_SETICON, ICON_BIG, (LPARAM)hicon);
 	} else {
 		icon = Ref<Image>();
-		SendMessage(windows[MAIN_WINDOW_ID].hWnd, WM_SETICON, ICON_SMALL, NULL);
-		SendMessage(windows[MAIN_WINDOW_ID].hWnd, WM_SETICON, ICON_BIG, NULL);
+		SendMessage(windows[MAIN_WINDOW_ID].hWnd, WM_SETICON, ICON_SMALL, (LPARAM)NULL);
```

Comment:
LPARAM is an integer not a pointer, so it is better to use `0` instead of NULL cast.

Reply:
I was concerned about doing that in case the `NULL` was there for a reason, but it may've just been a mistake